cns/client_test.go

```diff
@@ -1180,6 +1180,26 @@ func TestClient(t *testing.T) {
 			}
 		}
 
+		// Test BackingDiskObjectId field
+		var queryFilterBackingDiskObjectIdTest cnstypes.CnsQueryFilter
+		var volumeIDListBackingDiskObjectIdTest []cnstypes.CnsVolumeId
+		volumeIDListBackingDiskObjectIdTest = append(volumeIDListBackingDiskObjectIdTest, cnstypes.CnsVolumeId{Id: volumeID})
+		queryFilterBackingDiskObjectIdTest.VolumeIds = volumeIDListBackingDiskObjectIdTest
+		t.Logf("Calling QueryVolume using queryFilter: %+v", pretty.Sprint(queryFilterBackingDiskObjectIdTest))
+		queryResultBackingDiskObjectIdTest, err := cnsClient.QueryVolume(ctx, queryFilterBackingDiskObjectIdTest)
+		if err != nil {
+			t.Errorf("Failed to query volume. Error: %+v \n", err)
+			t.Fatal(err)
+		}
+		t.Logf("Successfully Queried Volumes. queryResult: %+v", pretty.Sprint(queryResult))
+		t.Log("Checking backingDiskObjectId retieved")
```

Comment:
Any reason why I didn't see this log message dumped in your test result?

Reply:
Placed this test inside a wrong block. Updated.